core/storage/feedback/gae_models.py

```diff
@@ -93,22 +93,37 @@ def get_deletion_policy():
         """General feedback thread needs to be pseudonymized for the user."""
         return base_models.DELETION_POLICY.LOCALLY_PSEUDONYMIZE
 
+    @staticmethod
+    def get_export_method():
+        """Model is exported as multiple unshared instances."""
```

Comment:
Ditto, here and elsewhere that is not "single unshared instance" (which is the natural default). I.e. try to explain why that natural default can't be used, and a different method had to be chosen.

Reply:
done